content/zh/docs/tutorials/stateful-application/cassandra.md

```diff
@@ -214,19 +214,19 @@ kubectl apply -f cassandra-statefulset.yaml
 
 1.获取 Cassandra StatefulSet:
```

Comment:
good catch!
the sorted list should have a blank symbol after the dot, please modify these by the way~
Thanks 🍭

Reply:
Amended in the latest commit

Reply:
please squash commits into one 😄